### ISSUE TITLE
Show song info in window title

### ISSUE THREAD

**Issue by efermi** [CDrummond/cantata#1104]
If it's possible how do I enable it, can't find in the settings. Use case: I use classic task bar showing window titles and maximized windows and like to see what's going on without switching windows or hovering over the taskbar.

**Comment by CDrummond** [comment]
Cantata used to do this, but this was removed in v1.4 The reason is that it duplicates info that is in the toolbar. So, you had the same information duplicated **very** close to each other, and I did not like this.

Using it in the taskbar, you're not going to see much information - unless you have really wide taskbar buttons. For example, if Cantata is configured for multiple MPD servers, it shows the current connection name in the title bar - so you would have "Cantata (Connection) - Song by Artist". This is quite large, and you'd probably only see "Cantata (Connection) - So..." in the taskbar.

Sorry, just not worth the hassle.
